etc/proto/run

```diff
@@ -1,7 +1,23 @@
-#!/bin/sh
+#!/bin/bash
 
 tar xf /dev/stdin
 
+# Make sure tools are all up-to-date so that our compiled protobufs aren't
+# generated by stale tools
+max_age="$(date --date="1 week ago" +%s)"
+if [[ "${max_age}" -gt "$(cat /last_run_time)" ]]; then
+  PRE="\e[1;31m~\e[0m"
+  echo -e "\e[1;31m~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\e[0m" >/dev/stderr
+  echo -e "${PRE} \e[1;31mWARNING:\e[0m pachyderm_proto is out of date" >/dev/stderr
+  echo -e "${PRE} please run" >/dev/stderr
+  echo -e "${PRE}   docker ps -a -q | xargs docker rm  "\#" rm all docker containers" >/dev/stderr
```

Comment:
Why remove all containers? This would probably kill a kubernetes cluster that's running in containers.

Reply:
After `make proto` finishes running, the container that it was running in is still around, it's just stopped (and you need to delete the container to delete the `pachyderm_proto` image, and you need to delete the image for rebuilding the image to re-download the protobuf compiler).

I couldn't figure out a way to delete only the `pachyderm_proto` container, but if you know how to do that (maybe with `docker ps --filter`, which I briefly tried to figure out but couldn't?) I agree that that would be better

Reply:
Just figured it out, I think